### PR TITLE
[DEPRECATIONS] stew.find

### LINKED ISSUE #122: find is just strange, I think we should deprecate
In nearly all cases, `stew.find` is better served by just using `Funnel`. I think it was a good experiment. We should confirm the above, and then find a good path forward.

### PR DESCRIPTION
**Pull request by locks**
Fixes https://github.com/stefanpenner/broccoli-stew/issues/122.

## Tasks
- [ ] Add deprecation method when calling `stew.find`
- [ ] Document deprecation in DEPRECATIONS.md

### CODE REVIEW

File: DEPRECATIONS.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Deprecations
+
+## v1.6.0
+
+### stew.find
+
+(insert technical rationale that I am not fully aware of)
```
Comment (stefanpenner) [inline DEPRECATIONS.md]: `stew.find(path)` tried to do magic to figure out what the user wanted, the result was a more confusing and brittle experience.

File: DEPRECATIONS.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Instead, you should use `broccoli-funnel`.
```
Comment (stefanpenner) [inline DEPRECATIONS.md]: I thought our discussion was that we merely deprecate `stew.find(x)` and allow `stew.find(x, { /* funnelOptions */)`

Comment (locks) [reply DEPRECATIONS.md]: Confirm, the PR was previous to the conversation :) I'll update tomorrow!

Comment (stefanpenner) [reply DEPRECATIONS.md]: ah ok 👍

**Pull request closed without merging**